test/onionshare_gui_test.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import os, sys, unittest, inspect, time
+from PyQt5 import QtCore, QtWidgets, QtGui, QtTest
+
+sys.path.append('../onionshare')
+sys.path.append('../onionshare_gui')
```

Comment:
I know these appends are ugly but I could not get the test to run otherwise, unless the test file itself was *also* in the root of the repo